File: c_projects/compare.c
#include <stdio.h>
/**
 * main - entry point of the program
 * description: a program to compare two numbers using else if
 *
 * Return: always (0) success
 */
int main(void)
{
	int num1, num2;

	printf("enter the first number\n");
	scanf("%d", &num1);
	printf("enter the second number\n");
	scanf("%d", &num2);
	if (num1 == num2)
	{
		printf("both numbers are equal\n");
	}
	else if (num1 > num2)
	{
		printf("the larger number is %d\n", num1);
		printf("the lesser number is %d\n", num2);
	}
	else
	{
		/*num1 < num2*/
		printf("the larger number is %d\n", num2);
		printf("the smaller number is %d\n", num1);
	}
	return (0);
}

File: interview/compare.py
# compare two numbers using else-if statement and output smaller and
# larger numbers in python
# Taking user input for two numbers
num1 = float(input("Enter the first number: "))
num2 = float(input("Enter the second number: "))

# Comparing the numbers using if-elif-else
if num1 > num2:
    print(f"Larger number: {num1}")
    print(f"Smaller number: {num2}")
elif num1 < num2:
    print(f"Larger number: {num2}")
    print(f"Smaller number: {num1}")
else:
    print("Both numbers are equal.")

